src/padreon/SimpleCape/Main.php

```diff
@@ -128,15 +128,15 @@ public function setCape(Player $player, string $cape, CommandSender $sender = nu
         $newSkin = new Skin($oldSkin->getSkinId(), $oldSkin->getSkinData(), $cape, $oldSkin->getGeometryName(), $oldSkin->getGeometryData());
         $player->setSkin($newSkin);
         $player->sendSkin();
-        if (!$cape == null) {
-            if (!$sender == null) {
+        if (!$cape === null) {
```

Comment:
still evaluates to false. $cape !== null

Reply:
ah thank you so much for review

Reply:
Next time i might be able to PR actually